src/main/navigation/navigation.c

```diff
@@ -143,6 +143,7 @@ PG_RESET_TEMPLATE(navConfig_t, navConfig,
         .max_dive_angle = 15,                   // degrees
         .cruise_throttle = 1400,
         .cruise_speed = 0,                      // cm/s
+        .control_smoothness = 0,
```

Comment:
One more thing - since you're adding new parameter you also need to increase PG version so the config for that PG is reset and not loaded from the incorrect data in flash.

This should be sufficient:
`PG_REGISTER_WITH_RESET_TEMPLATE(navConfig_t, navConfig, PG_NAV_CONFIG, 7);`

Reply:
I was actually wondering how that was handled! Thanks!